Source/WebKit/UIProcess/API/Cocoa/_WKContextMenuElementInfo.mm

```diff
@@ -24,12 +24,22 @@
  */
 
 #import "config.h"
-#import "_WKContextMenuElementInfo.h"
+#import "_WKContextMenuElementInfoInternal.h"
 
 #if !PLATFORM(IOS_FAMILY)
 
 @implementation _WKContextMenuElementInfo
 
+- (instancetype)_initWithHitTestResult:(_WKHitTestResult *)hitTestResult
+{
+    if (!(self = [super init]))
+        return nil;
+
+    _hitTestResult = hitTestResult;
```

Comment:
I'm not sure this properly retains and releases the _WKHitTestResult to give it the right lifetime.  Implicit stuff like this in non-ARC ObjC makes me a little scared.

Reply:
I think I've addressed this now that I'm using an API object.